### PR TITLE
[SC-340] Mar 20 Spell

### PR DESCRIPTION
**Pull request by barrutko**
**This spell (and tests) assumes that the previous spell will be executed first (06 Mar)**

### CODE REVIEW

File: src/interfaces/IOwnable.sol [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.10;
+
+interface IOwnable {
+    function acceptOwnership() external;
+    function owner() external view returns (address);
+    function pendingOwner() external view returns (address);
+    function transferOwnership(address newOwner) external;
+}
```
Comment (hexonaut) [inline src/interfaces/IOwnable.sol:9]: For this one-off stuff we can just inline it.

File: src/proposals/20240320/SparkEthereum_20240320.sol [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import { SparkPayloadEthereum } from '../../SparkPayloadEthereum.sol';
+
+/**
+ * @title  March 20, 2024 Spark Ethereum Proposal - Raise maximum supply cap in Cap Automator to 6,000 for WBTC and adjust DAI spread to 14% target APY
+ * @author Phoenix Labs
+ * @dev    This proposal changes the max parameter in supplyCapConfig in CapAutomator for WBTC market
+ * Forum:  https://forum.makerdao.com/t/mar-6-2024-proposed-changes-to-sparklend-for-upcoming-spell/23791
+ *         https://forum.makerdao.com/t/stability-scope-parameter-changes-11-under-sta-article-3-3/23910
+ * Votes:  TODO
```
Comment (hexonaut) [inline src/proposals/20240320/SparkEthereum_20240320.sol:15]: Votes are up:

https://vote.makerdao.com/polling/QmVGDsvm
https://vote.makerdao.com/polling/QmYYoAMe

Forum post about the Morpho vault will be posted by BA soon. They can do changes without polling on this.

**Review by hexonaut — CHANGES_REQUESTED** [review]

**Review by hexonaut — DISMISSED** [review]
Good to deploy.

**Review by hexonaut — CHANGES_REQUESTED** [review]
Tested spell locally, and it's good. Sending off to govops team.

CI is failing though we need to remove the lib/ that was added.

**Review by hexonaut — APPROVED** [review]
Ok LGTM

**Review by barrutko — COMMENTED** [review]
Reviewed changes added on top of my commits. LGTM.

**Comment by hexonaut** [comment]
Merging as we had to hand off the spell to @lucas-manuel part way.

**Pull request merged**